routes/drones.js

```diff
@@ -1,37 +1,52 @@
-const express = require('express');
+const express = require("express");
 
-// require the Drone model here
+const Drone = require("../models/Drone.model");
 
 const router = express.Router();
 
-router.get('/drones', (req, res, next) => {
-  // Iteration #2: List the drones
-  // ... your code here
+router.get("/drones", (req, res, next) => {
+  Drone.find({})
+    .then((dronesFromDB) => {
+      res.render("drones/list", { drones: dronesFromDB });
+    })
+    .catch((error) => {
+      console.log(error);
+    });
 });
 
-router.get('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+router.get("/drones/create", (req, res, next) => {
+  res.render("drones/create-form");
 });
 
-router.post('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+router.post("/drones/create", (req, res, next) => {
+  const { name, propellers, maxSpeed } = req.body;
+  Drone.create({ name, propellers, maxSpeed })
+    .then(() => res.redirect("/drones"))
+    .catch((error) => console.log(error), res.render("drones/create-form"));
 });
 
-router.get('/drones/:id/edit', (req, res, next) => {
-  // Iteration #4: Update the drone
-  // ... your code here
+router.get("/drones/:id/edit", (req, res, next) => {
+  const { id } = req.params;
+
+  Drone.findById(id).then((editDrone) => {
+    res.render("drones/update-form", editDrone);
+  });
```

Comment:
You forgot to add a `.catch(error => console.log(error))`. It always must be after `.then()` in case of an error